src/ResponsiveAnalogRead.cpp

```diff
@@ -36,6 +42,19 @@ ResponsiveAnalogRead::ResponsiveAnalogRead(int pin, bool sleepEnable, float snap
   setSnapMultiplier(snapMultiplier);
 }
 
+
+void ResponsiveAnalogRead::begin(int pin, bool sleepEnable, float snapMultiplier){
+    pinMode(pin, INPUT ); // ensure button pin is an input
```

Comment:
Setting the pin to `INPUT` (as opposed to `INPUT_PULLUP`) disables the internal pullup, which makes the next statement redundant.

Reply:
I didn't know this, thanks.